packages/elasticsearch-plugin/src/options.ts

```diff
@@ -165,7 +165,7 @@ export interface ElasticsearchOptions {
      * @description
      * Custom mappings may be defined which will add the defined data to the
      * Elasticsearch index and expose that data via the SearchResult GraphQL type,
-     * adding a new `customMappings` field.
+     * adding a new `customMappings` field, customProductMappings, customProductVariantMappings.
```

Comment:
should read:

"adding new `customMappings`, `customProductMappings` & `customProductVariantMappings` fields"